sdk/helper/pluginutil/run_config.go

```diff
@@ -20,6 +20,15 @@ import (
 	"github.com/hashicorp/vault/sdk/helper/pluginruntimeutil"
 )
 
+const (
+	// Labels for plugin container ownership
+	LabelVaultPID           = "com.hashicorp.vault.pid"
+	LabelVaultClusterID     = "com.hashicorp.vault.clusterID"
+	LabelVaultPluginName    = "com.hashicorp.vault.pluginName"
+	LabelVaultPluginVersion = "com.hashicorp.vault.pluginVersion"
+	LabelVaultPluginType    = "com.hashicorp.vault.pluginType"
```

Comment:
I think these can stay unexported:

```suggestion
	labelVaultPID           = "com.hashicorp.vault.pid"
	labelVaultClusterID     = "com.hashicorp.vault.clusterID"
	labelVaultPluginName    = "com.hashicorp.vault.pluginName"
	labelVaultPluginVersion = "com.hashicorp.vault.pluginVersion"
	labelVaultPluginType    = "com.hashicorp.vault.pluginType"
```

Reply:
Done in 6a8700b759